angular-spa-ui/js/search-app/tags-module/index.js

```diff
@@ -2,16 +2,35 @@
 
 module.exports = function (angular) {
 
+    var tagsCtrl = require('./tags-ctrl');
+
     var tags = angular.module('app.tags', []);
-    
-    tags.config(configCb);
-    
+
+    tags
+        .config(configCb)
+        .directive('tagIteme', tagIteme);
+
     function configCb($stateProvider) {
         $stateProvider
             .state('tags', {
                 url: '/tags',
-                template: 'Hello tags'
-            })
+                templateUrl: 'tagsTemplate.html',
+                controller: tagsCtrl
+            });
     };
 
+    function tagIteme() {
+        return {
+            restrict: 'E',
+            templateUrl: 'tagItemTemplate.html',
+            link: function(scope, element, attrs) {
+                //console.log(scope)
```

Comment:
Commented code in pull request allowed in next situations:
1. That is a feature for future
2. That is a feature that should be disabled at the moment for some time
3. That is a quick fix for immediately testing

Please remove commented code